docs/Homebrew-Governance.md

```diff
@@ -66,44 +62,60 @@
 
 ## 5. Meetings of the Project Leadership Committee
 
-1. A meeting of the PLC may be called by any two of its members with at least three weeks notice, unless all PLC members agree to a shorter notice period.
+1. A synchronous meeting of the PLC may be called by any two of its members with at least three weeks notice, unless all PLC members agree to a shorter notice period.
 
-2. The quorum to vote on resolutions at a meeting of the PLC is a majority of its members.
+2. The quorum to vote on resolutions at a synchronous meeting of the PLC is a majority of its members. In a Slack vote, there a time limit instead of quorum: it will take effect after a week, assuming vote passes.
```

Comment:
Not a blocker, but if anyone on @Homebrew/plc can clarify this a bit (see https://github.com/Homebrew/homebrew-governance-private/pull/28#discussion_r767067693) before the AGM, please do.

Reply:
That comment was:

> Perhaps could define when a meeting or a Slack vote should be used. Point 1 describes how a meeting is organised but there's no info here about a Slack vote.

Reply:
Can still answer questions here although the wording won't change now that people have started voting?